integrations/va-auth0/src/index.tsx

```diff
@@ -225,9 +225,19 @@ const handleFetchEvent: FetchEventCallback<Auth0RuntimeContext> = async (request
                             );
                         }
                     } else {
-                        return new Response('Error: No Access Token found in response from Auth0', {
-                            status: 401,
-                        });
+                        logger.debug(
+                            `Did not receive access token. Error: ${(resp && resp.error) || ''} ${
+                                (resp && resp.error_description) || ''
+                            }`
+                        );
+                        return new Response(
+                            `Error: No Access Token found in response from Auth0 ${
+                                (resp && resp.error) || ''
+                            } ${(resp && resp.error_description) || ''}`,
```

Comment:
Have you verified that error.description won't possibly leak any keys into the logs?

Reply:
These errors are returned by the auth providers, so they're unlikely to contain any keys. However, there is no way to confirm that. At least I can't think of a way to confirm that

Reply:
I agree with @scazan. We should not surface errors directly as it could contain sensitive information.

Reply:
@taranvohra but is it fine to log them? And can we surface just response.error to the user (and not response.error_description)

Reply:
Logs are fine because only the integration owner can see them.

Reply:
Fixed!